.github/workflows/ci.yaml

```diff
@@ -66,18 +70,17 @@ jobs:
       run: |
         micromamba install openmm -c conda-forge
 
-    - name: Uninstall OpenMM
-      if: ${{ matrix.openmm == false && matrix.openeye == true }}
+    - name: Install Python <3.13 dependencies
+      if: ${{ matrix.python-version != '3.13' }}
       run: |
-        micromamba remove openmm mdtraj
-        # Removing mBuild also removes some leaves, need to re-install them
-        micromamba install rdkit packmol "lammps >=2023.08.02"
```

Comment:
(tangent) No longer relevant but I think `micromamba remove --force` would leave the deps

Reply:
I recall that not working (which was a surprise to me) for reasons that were more frustrating to dig into than work around